docs/concepts/sys-update.md

```diff
@@ -0,0 +1,65 @@
+---
+sidebar_position: 9
+sidebar_label: "System Updates"
+title: About System Updates
+description: Update Treasury functionality including contract, policy, and app widget.
+---
+
+## About updates
+
+The [system update](../settings.md#system-updates) functionality allows Treasury developers to provide updates such as contract upgrades, policy changes, or instance app widget changes to treasury instances.
+These available updates can be reviewed, approved, and applied manually by treasury administrators.
+
+## Update Types
+
+The following system updates are supported:
+
+- **Web4 Contract**: Used to upgrade the underlying Web4 smart contract for the treasury instance to the latest version. Applying this update triggers a self-upgrade of the contract.
+
+- **Widgets**: Updates the main application widget to the latest version from the reference account. Ensures the UI and logic are up-to-date with the latest features and bug fixes.
+
+- **Policy**: Updates that change the DAO policy, such as voting rules, roles, or permissions. Applying this update creates a proposal to change the policy on the DAO contract.
+
+- **DAO contract**: Upgrades the DAO contract itself (e.g., to a new version of Sputnik DAO). May involve deploying new contract code or migrating state.
+
+
+## How System Updates Work
+
+When a system update is published, treasury administrators may see a notification banner or badge indicating that a new update is available for their treasury instance. This is designed to help administrators and users stay informed about important changes, such as contract upgrades or policy updates.
+
+### Banner Notifications
+
+When an update is available and has not been marked as finished in your browser, a banner or notification will appear:
```

Comment:
when the user closes it (clicks on cancel), it will not appear again.

Reply:
@petersalomonsen so if I closes it and refresh it will show again? I thought we discussed it in group call that once the user discard it we don't show it anymore?